test/setup_kind.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+GO111MODULE="on" go get sigs.k8s.io/kind@v0.6.0
+export PATH=$(go env GOPATH)/bin:$PATH
+
+kind delete cluster --name=e2e &> /dev/null || true
+
+if ! (kind create cluster --name=e2e) ; then
```

Comment:
in CI we should delete the kind cluster somewhere in the test runner, preferably in an exit trap so we ensure cleanup.

The docker in docker stuff in CI will try to cleanup, but it's better if the job cleans up 😅

Reply:
https://github.com/kubernetes-sigs/kind/blob/3b0b767255a382df846377f9287444268fec1825/hack/ci/e2e-k8s.sh#L157

https://github.com/kubernetes-sigs/kind/blob/3b0b767255a382df846377f9287444268fec1825/hack/ci/e2e-k8s.sh#L28-L38